Format /web/api/[r]* using Prettier

queengooborg opened the pull request:
This PR is a part of a project to format all of the documents using Prettier, and then enforce Prettier formatting for the files going forward.  This PR formats the `/web/api` folder for APIs starting with R.

queengooborg committed:
Format /web/api/[r]* using Prettier

queengooborg committed:
Merge branch 'main' into prettier/web/api/r

queengooborg committed:
Merge branch 'main' into prettier/web/api/r

queengooborg committed:
Merge branch 'main' into prettier/web/api/r

queengooborg committed:
Merge branch 'main' into prettier/web/api/r

queengooborg committed:
Merge branch 'main' into prettier/web/api/r

queengooborg committed:
Merge branch 'main' into prettier/web/api/r

bsmth committed:
Merge branch 'main' into prettier/web/api/r

bsmth commented:
Taking a look 👀

bsmth commented on files/en-us/web/api/rtcdatachannel/bufferedamountlow_event/index.md:
```diff
@@ -32,15 +32,20 @@ This example sets up a handler for `bufferedamountlow` to request more data any
-    dc.send(source.readFile(65536));
-  }
-}, false);
+let source =
+  /* source data object */
+
+  (dc.bufferedAmountLowThreshold = 65536);
```
```suggestion
// source data object
let source = (dc.bufferedAmountLowThreshold = 65536);
```

bsmth commented on files/en-us/web/api/rtcpeerconnection/icegatheringstatechange_event/index.md:
```diff
@@ -38,32 +38,36 @@ This example creates a handler for `icegatheringstatechange` events.
+
+    switch (connection.iceGatheringState) {
+      case "gathering":
+        /* collection of candidates has begun */
+        break;
+      case "complete":
+        /* collection of candidates is finished */
```
Looks like we'll need to tackle a lot of misuse of single vs multi-line comments in general.

```suggestion
        // collection of candidates has begun
        break;
      case "complete":
        // collection of candidates is finished
```

bsmth commented on files/en-us/web/api/rtcpeerconnection/negotiationneeded_event/index.md:
```diff
@@ -33,17 +33,23 @@ A generic {{domxref("Event")}}.
+        })
+      )
+      .catch((err) => {
+        /* handle error */
```
```suggestion
        // handle error
```

bsmth commented on files/en-us/web/api/rtcpeerconnection/negotiationneeded_event/index.md:
```diff
@@ -53,14 +59,16 @@ You can also set an event handler for the `negotiationneeded` event by assigning
+      })
+    )
+    .catch((err) => {
+      /* handle error */
```
```suggestion
      // handle error
```

bsmth commented on files/en-us/web/api/rtcremoteoutboundrtpstreamstats/localid/index.md:
```diff
@@ -127,13 +127,26 @@ async function networkTestStop(pc) {
+            "remoteId",
+            endRemoteOutbound.id
+          );
+
+          const elapsedTime =
+            (endRemoteOutbound.timestamp - startRemoteOutbound.timestamp) /
+            1000; /* in seconds */
```
```suggestion
          // Elapsed time in seconds
          const elapsedTime =
            (endRemoteOutbound.timestamp - startRemoteOutbound.timestamp) /
            1000;
```

bsmth committed:
Apply suggestions from code review

Taking the opportunity to change some multi-line comments to single-line where relevant

bsmth committed:
Update files/en-us/web/api/rtcremoteoutboundrtpstreamstats/localid/index.md

bsmth commented on files/en-us/web/api/readablestreambyobreader/closed/index.md:
```diff
@@ -20,10 +20,14 @@ A {{jsxref("Promise")}}.
-  .then(() => { /* Resolved - code to handle stream closing */ } )
-  .catch(() => { /* Rejected - code to handle error */ } );
+  .then(() => {
+    /* Resolved - code to handle stream closing */
+  })
+  .catch(() => {
+    /* Rejected - code to handle error */
```
```suggestion
    // Resolved - code to handle stream closing
  })
  .catch(() => {
    // Rejected - code to handle error
```

bsmth commented on files/en-us/web/api/readablestreambyobreader/index.md:
```diff
@@ -92,8 +95,12 @@ The {{domxref("ReadableStreamBYOBReader.closed")}} property returns a promise th
-  .then(() => { /* Resolved - code to handle stream closing */ } )
-  .catch(() => { /* Rejected - code to handle error */ } );
+  .then(() => {
+    /* Resolved - code to handle stream closing */
+  })
+  .catch(() => {
+    /* Rejected - code to handle error */
```
```suggestion
    // Resolved - code to handle stream closing
  })
  .catch(() => {
    // Rejected - code to handle error
```

bsmth committed:
Apply suggestions from code review

Some multi-line comments changes to single-line, the rest should be handled in a follow-up.

bsmth reviewed (approved):
Thanks @queengooborg going to approve and merge. We need to handle some cases of multi-line comments vs single line comments in a follow-up.
